src/app/public/modules/adapter-service/adapter.module.ts

```diff
@@ -0,0 +1,14 @@
+import {
+  NgModule
+} from '@angular/core';
+
+import {
+  SkyCoreAdapterService
+} from './adapter.service';
+
+@NgModule({
+  providers: [
+    SkyCoreAdapterService
+  ]
+})
+export class SkyAdapterModule { }
```

Comment:
I think we should rename the module here too.

Reply:
Ah, good catch. Done.